src/components/Button.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import "../components/Buttons.css";
+
+export default function Button(props) {
+  // let returnElement = undefined
+
+  // if (props.type === "submit") {
+  //     returnElement = <input type="submit"></input>
+  // } else {
+  //     returnElement= <button className={props.className || "button"} {...props} />;
+  // }
```

Comment:
pls no commented-out code unless you are taking notes in a WIP or writing a tutorial or exercise